prac_08/silver_service_taxi.py

```diff
@@ -0,0 +1,20 @@
+"""Practical 08 - Jacob Finger - SilverServiceTaxi Class"""
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Version of Taxi with flagfall and fare cost that is multiplied by a given fanciness factor"""
+    flag_fall = 4.5
+
+    def __init__(self, name, fuel, fanciness):
+        """Initialise a SilverServiceTaxi instance, based on parent class Taxi"""
+        super().__init__(name, fuel)
+        self.price_per_km = Taxi.price_per_km * fanciness
```

Comment:
Another way of multiplying Price_per_km by fanciness would be:
self.price_per_km *= fanciness.

Reply:
Sleek, I like it